package.json

```diff
@@ -14,10 +14,14 @@
         "build:android": "npm run build && npx cap sync android && npx cap copy android && npm run open:android",
         "open:android": "npx cap open android",
         "run:android": "npm run build:android && ionic capacitor run android --livereload --external",
-        "release:all:prod": "npm run tag:before:prod && npm run build && firebase use instantvisio && firebase deploy && npm run tag:after",
-        "release:all:dev": "npm run tag:before:dev && firebase use instantvisio-dev &&  env-cmd -f .env.development npm run build && firebase deploy && npm run tag:after",
-        "release:hosting:prod": "npm run tag:before:prod && npm run build && firebase use instantvisio && firebase deploy --only hosting && npm run tag:after",
-        "release:hosting:dev": "npm run tag:before:dev && firebase use instantvisio-dev && env-cmd -f .env.development npm run build && firebase deploy --only hosting",
+        "deploy:all:prod": "npm run build && firebase use instantvisio && firebase deploy",
+        "deploy:all:dev": "firebase use instantvisio-dev &&  env-cmd -f .env.development npm run build && firebase deploy",
+        "release:all:prod": "npm run tag:before:prod && npm run deploy:all:prod && npm run tag:after",
+        "release:all:dev": "npm run tag:before:dev && npm run deploy:all:dev && npm run tag:after",
+        "deploy:hosting:prod": "npm run build && firebase use instantvisio && firebase deploy --only hosting",
```

Comment:
can't you reuse `deploy:all:prod` here and pass the `--only hosting` flag at the end of the command ? I think `npm` allows that by using `-- -hosting`